ANSWERS.md

```diff
@@ -0,0 +1,19 @@
+1. What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
+
+Proptypes are used to check data being passed down via props. It helps catch errors and gives you a better Idea of what the errors mean.
+
+2. Describe a life-cycle event in React?
+
+A function that gives us the ability to mount and unmount a component. This way you store it on local storage to be called back after a page refresh.
```

Comment:
You wouldn't store a life-cycle event in local storage. Check Tyler Thompson's definition of a life-cycle event: 

```
A life-cycle event is one of a number of phases that a component goes through as it is being mounted, updated, and eventually unmonunted from a page. It starts with the mounting phase in which a render method is called and the component is being built. Then it moves to the updating phase where setState can be used to update the data and force a re-render. Finally there is unmounting when the component is removed from the screen.
```